packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -159,6 +159,7 @@ const styleSheets = {
             flexDirection: "column",
             background: "white",
             boxSizing: "border-box",
+            overflow: "hidden",
```

Comment:
Were the corners not being rounded correctly?

Reply:
Exactly, Nefaur noticed that and I found out this was the best place to apply the fix.